airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/DateTimeConverter.java

```diff
@@ -82,7 +102,15 @@ public static String convertToTimestamp(final Object timestamp) {
     } else if (timestamp instanceof final Instant i) {
       // Incremental mode
       return resolveEra(i.atZone(UTC).toLocalDate(), i.atOffset(UTC).toLocalDateTime().format(TIMESTAMP_FORMATTER));
+    } else if (timestamp instanceof final LocalDateTime localDateTime) {
+      final LocalDate date = localDateTime.toLocalDate();
+      final String value = localDateTime.format(TIMESTAMP_FORMATTER);
+      return resolveEra(date, value);
     } else {
+      if (!loggedUnknownTimestampClass) {
```

Comment:
The reason I only want to log it once is to avoid the scenario where we have a million of these logs! That could happen in case if a table has a million records and debezium starts sending a different class type for those records